resources/js/lang/de/rooms.js

```diff
@@ -162,7 +164,8 @@ export default {
         disabled: 'Deaktiviert',
         enabled: 'Aktiviert',
         onlyForGuestsEnabled: 'Aktiviert für Gäste'
-      }
+      },
+      recordAttendance: 'Anwesenheit protokollieren'
```

Comment:
Maybe `Anwesenheit der Teilnehmer protokollieren`. Since it is not only the logs for the current user!